src/libs/elektra/kdb.c

```diff
@@ -957,7 +957,6 @@ int kdbGet (KDB * handle, KeySet * ks, Key * parentKey)
 
 	if (ns == KEY_NS_EMPTY)
 	{
-		/*TODO: Solution ("Please use the cascading key / instead")*/
 		ELEKTRA_ADD_VALIDATION_SYNTACTIC_WARNING (parentKey, "Empty namespace passed to kdbGet");
```

Comment:
Simply add the solution here (as it is the fix to avoid the warning).

And the category is wrong: it is a wrong use of API and has nothing to do with validation of configuration.